docs/eventing/samples/writing-event-source/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Writing an Event Source"
+title: "Writing an Event Source controller"
```

Comment:
Same comment.

Reply:
For all of these I'd change Event Source => event source

Idk about what to put for the titles, @lionelvillard is it worth blocking the PR or can we figure these out later if they need to be updated? Otherwise can you please add suggestions for what you think would be good titles?

Reply:
Side note: we're not really using this include thing with the readme anymore afaik, so the content could be moved into index directly if you don't mind @matzew